HHS/Head-Start-TTADP
# [TTAHUB-199] add all regions to activity report dropdown if user has central office

## Description of change

Added "All Regions" to the region select dropdown on the activity-reports page if the user has "central office" as their home region.

## How to test

Navigate to the landing page as a user with central office (14) as their home region. Notice that "All regions" is present in the dropdown. Switch between regions. The alerts and reports should change for that region. When All Regions are selected, the new report button will not appear.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-199

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
